lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,154 @@
+
+module Scrabble
+
+	LETTER_VALUES = [
+		[1, "a", "e", "i", "o", "u", "l", "n", "r", "s", "t"],
+		[2, "d", "g"],
+		[3, "b", "c", "m", "p"],
+		[4, "f", "h", "v", "w", "y"],
+		[5, "k"],
+		[8, "j", "x"],
+		[10, "q", "z"]
+	]
+	
+
+	class Scrabble
+
+		def self.value_of(letter)
+			LETTER_VALUES.each do |value_letters_array|
+				if value_letters_array.include?(letter)
+					return value_letters_array[0]
+				end
+			end
+		end
+
+		def self.score(word)
+			word = word.to_s.downcase
+
+			word_score = 0
+
+			word_array = word.chars.to_a
```

Comment:
`.chars` returns an array, I'm pretty sure the `.to_a` is redundant, unless there is an edge case I'm not thinking of!